scripting/get5/matchconfig.sp

```diff
@@ -175,14 +175,24 @@ bool LoadMatchConfig(const char[] config, bool restoreBackup = false) {
         }
       }
     }
+    CreateTimer(1.1, Timer_SetClanTagsAfterMatchLoad, _, TIMER_FLAG_NO_MAPCHANGE);
```

Comment:
Any specific reason for setting this to 1.1 seconds?

Reply:
The timer placing players is 1 second. See above.

Reply:
Thanks GitHub for shrinking the diff to avoid the giant comment block that would help make more sense of this :) okay perfect! LGTM!